helm/prometheus-rules/values.schema.json

```diff
@@ -2,6 +2,42 @@
     "$schema": "http://json-schema.org/schema#",
     "type": "object",
     "properties": {
+        "Installation": {
+            "type": "object",
+            "properties": {
+                "V1": {
+                    "type": "object",
+                    "properties": {
+                        "Guest": {
+                            "type": "object",
+                            "properties": {
+                                "Kubernetes": {
+                                    "type": "object",
+                                    "properties": {
+                                        "IngressController": {
+                                            "type": "object",
+                                            "properties": {
+                                                "BaseDomain": {
+                                                    "type": "string"
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
```

Comment:
Can you remove this please ?

Reply:
No because it is in the values for vintage

Reply:
Yes and it's only use in 3 installations actually 😂